### DISCUSSION TITLE
ExprTooDeep error

### DISCUSSION THREAD

**Discussion by rosscanning** [rhaiscript/rhai#575] [Q&A] [answered]
This compiles:
```
    let tmp = ["Option A"];
    msg_box("Message", tmp);
```
but this fails to compile with ExprTooDeep.
```
     msg_box("Message", ["Option A"]);
```
Is this a limitation of the compiler, and are these limitation documented anywhere?

Thanks

**Comment by schungx** [comment, marked as answer]
https://rhai.rs/book/safety/max-stmt-depth.html

**Comment by schungx** [comment]
Closing this for now. Feel free to reopen if necessary.
